### PR TITLE
Fix Bug 1307091 - Change the redirect from en-US/firefox/nightly/ to the new pages

### PR DESCRIPTION
**Pull request by kyoshino**

### CODE REVIEW

File: bedrock/firefox/redirects.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def firefox_mobile_faq(request, *args, **kwargs):
     redirect(r'^firefox/accountmanager/?$', 'https://developer.mozilla.org/Persona'),
 
     # bug 841846
```
Comment (alexgibson) [inline bedrock/firefox/redirects.py]: Nit: please append the bug number along with the existing one here so we have some paper trail :)

File: tests/redirects/map_globalconf.py [hunk 2]
```diff
@@ -797,7 +797,7 @@
     url_test('/persona{/{about,developer-faq}/,}', 'https://developer.mozilla.org/Persona'),
 
     # bug 841846
```
Comment (alexgibson) [inline tests/redirects/map_globalconf.py]: Same here please

File: bedrock/firefox/redirects.py (hunk 1 above)
Comment (kyoshino) [inline bedrock/firefox/redirects.py]: Yeah noticed that ;) Fixed!

**Comment by kyoshino** [comment]
Thanks!

**Pull request merged**